docs/table/access_table.rst

```diff
@@ -374,6 +374,49 @@ For columns the syntax and behavior of
   2973.0
   Length = 100 rows
 
+  Column alignment
+''''''''''''''''
```

Comment:
@agroener - Sure, no problem. I stared a lot of sphinx outputs when the docs was still full of build warnings, so now know what to look for :)